### PR TITLE
BZ2033580: corrected wrong instances of plug-in and parameter

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.10 and above versions
- [Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=2033580)
- [Preview](https://deploy-preview-43918--osdocs.netlify.app/openshift-enterprise/latest/welcome/index.html)
- **Scope** : The scope of this bz is to fix wrong instances of plug-in in content but **not in code/xref/titles.**
- This bz also fixes two instances of parameter which is wrongly spelled.

### CODE REVIEW

File: modules/builds-understanding-openshift-pipeline.adoc [hunk 1]
```diff
@@ -14,20 +14,20 @@ Jenkins images on {product-title} are fully supported and users should follow Je
 
 Pipelines give you control over building, deploying, and promoting your applications on {product-title}. Using a combination of the Jenkins Pipeline build strategy, `jenkinsfiles`, and the {product-title} Domain Specific Language (DSL) provided by the Jenkins Client Plug-in, you can create advanced build, test, deploy, and promote pipelines for any scenario.
 
-*{product-title} Jenkins Sync Plugin*
+*{product-title} Jenkins Sync Plug-in*
```
Comment (kalexand-rh) [inline modules/builds-understanding-openshift-pipeline.adoc:17]: @Preeticp, will you PTAL at the "plug-in" changes in this module? I can't find an official name for the plug-in, so I'm unsure if this capitalization change is correct.

File: modules/setting-up-vmc-for-vsphere.adoc [hunk 2]
```diff
@@ -73,7 +73,7 @@ It is recommended to move your vSphere cluster to the VMC `Compute-ResourcePool`
 
 [NOTE]
 ====
-You cannot use the VMware NSX Container Plugin for Kubernetes (NCP), and NSX is not used as the OpenShift SDN. The version of NSX currently available with VMC is incompatible with the version of NCP certified with {product-title}.
+You cannot use the VMware NSX Container Plug-in for Kubernetes (NCP), and NSX is not used as the OpenShift SDN. The version of NSX currently available with VMC is incompatible with the version of NCP certified with {product-title}.
```
Comment (kalexand-rh) [inline modules/setting-up-vmc-for-vsphere.adoc:76]: This aligns with [VMware's use](https://docs.vmware.com/en/VMware-NSX-T-Data-Center/3.1/ncp-kubernetes/GUID-52A92986-0FDF-43A5-A7BB-C037889F7559.html).

File: modules/builds-understanding-openshift-pipeline.adoc [hunk 3]
```diff
@@ -14,20 +14,20 @@ Jenkins images on {product-title} are fully supported and users should follow Je
...
+ * Synchronizing secrets into Jenkins credential entries {product-title} builds to the Jenkins git plug-in.
 
-*{product-title} Jenkins Client Plugin*
+*{product-title} Jenkins Client Plug-in*
 
-The {product-title} Jenkins Client Plugin is a Jenkins plugin which aims to provide a readable, concise, comprehensive, and fluent Jenkins Pipeline syntax for rich interactions with an {product-title} API Server. The plugin uses the {product-title} command line tool, `oc`, which must be available on the nodes executing the script.
+The {product-title} Jenkins Client Plug-in is a Jenkins plug-in which aims to provide a readable, concise, comprehensive, and fluent Jenkins Pipeline syntax for rich interactions with an {product-title} API Server. The plug-in uses the {product-title} command line tool, `oc`, which must be available on the nodes executing the script.
 
 The Jenkins Client Plug-in must be installed on your Jenkins master so the {product-title} DSL will be available to use within the `jenkinsfile` for your application. This plug-in is installed and enabled by default when using the {product-title} Jenkins image.
```
Comment (Preeticp) [inline modules/builds-understanding-openshift-pipeline.adoc:32]: This is not part of the PR but to be consistent can you change the Plug-in to Plugin, please?

**Review by Preeticp — COMMENTED** [review]
@Amrita42, thank you. I have left some suggestions, PTAL. Also, some information is missing, I have asked for inputs from Rolfe (current writer) and Brandi (earlier writer) for the same. If it is too much work for this PR, maybe we can create a follow-up issue and PR to track that work.

File: modules/builds-understanding-openshift-pipeline.adoc (hunk 1 above)
Comment (Preeticp) [reply modules/builds-understanding-openshift-pipeline.adoc:17]: @Amrita42 please hold on this change. Rolfe felt we should make an effort to ask the Developers if we could change the name to say plug-in. If that is possible then your changes would be good and you would not need to revert.

Comment (Amrita42) [reply modules/builds-understanding-openshift-pipeline.adoc:17]: @Preeticp Thanks for reviewing. As discussed I will hold off till you get an affirmation from dev. I think it's great if we can keep plug-in in sync with our style standards. Thanks @rolfedh for making that effort.

**Comment by rolfedh** [comment]
Based on the conversation that @Preeticp  and I had with the Builds team, we have standardized on "plug-in" (hyphenated), not "plugin." @Preeticp will confirm here as well.

File: modules/builds-understanding-openshift-pipeline.adoc (hunk 1 above)
Comment (Preeticp) [reply modules/builds-understanding-openshift-pipeline.adoc:17]: Hey @Amrita42, thanks to @rolfedh we have got acks from the devs to change it to the standard Plug-in, rather than Plugin. So your original changes can go in as is. Thank you, everyone, I'm happy we could agree on the standard way of writing the term.

Comment (Preeticp) [reply modules/builds-understanding-openshift-pipeline.adoc:17]: Here are the PRs created by Rolfe to change the upstream instances as well: https://github.com/openshift/jenkins-sync-plugin/pull/421/files and https://github.com/openshift/jenkins-client-plugin/pull/387/files (edited)

Comment (Amrita42) [reply modules/builds-understanding-openshift-pipeline.adoc:17]: Thank you everyone for this collabpration and I am glad we can have this consistency. Will update PR tday.

Comment (Preeticp) [reply modules/builds-understanding-openshift-pipeline.adoc:17]: I am resolving all my comments which are no longer relevant.

**Comment by Preeticp** [comment]
@rolfedh just wanted to bring to your notice this comment: https://github.com/openshift/openshift-docs/pull/44125#discussion_r842593825 It should not block this PR though. Maybe we could track it through another Jira.

File: modules/builds-understanding-openshift-pipeline.adoc [hunk 4]
```diff
@@ -14,20 +14,20 @@ Jenkins images on {product-title} are fully supported and users should follow Je
...
-The {product-title} Jenkins Sync Plugin keeps the build configuration and build objects in sync with Jenkins jobs and builds, and provides the following:
+The {product-title} Jenkins Sync Plug-in keeps the build configuration and build objects in sync with Jenkins jobs and builds, and provides the following:
 
  * Dynamic job and run creation in Jenkins.
  * Dynamic creation of agent pod templates from image streams, image stream tags, or config maps.
  * Injecting of environment variables.
  * Pipeline visualization in the {product-title} web console.
- * Integration with the Jenkins git plugin, which passes commit information from
- * Synchronizing secrets into Jenkins credential entries {product-title} builds to the Jenkins git plugin.
+ * Integration with the Jenkins git plug-in, which passes commit information from
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/builds-understanding-openshift-pipeline.adoc:25]: > @bmcelvee @rolfedh this point seems incomplete. Tagging you Brandi, in case you have any inputs, since you handed over this content to us.

Thanks, I've created https://issues.redhat.com/browse/RHDEVDOCS-3953 to fix this bug in a separate PR.

**Comment by rolfedh** [comment]
> @rolfedh just wanted to bring to your notice this comment: [#44125 (comment)](https://github.com/openshift/openshift-docs/pull/44125#discussion_r842593825) It should not block this PR though. Maybe we could track it through another Jira.

Thanks for highlighting this, @Preeticp. In response, I've created a Jira task to fix this bug, which I'll tackle right away. 

See https://github.com/openshift/openshift-docs/pull/44125#discussion_r843852755

**Comment by Amrita42** [comment]
@kalexand-rh ptal , i think there are further changes required. Thanks.
